lib/phoenix_live_component.ex

```diff
@@ -330,8 +330,8 @@ defmodule Phoenix.LiveComponent do
 
   The above approach is the preferred one when passing blocks to `do/end`.
   However, if you are outside of a .leex template and you want to invoke a
-  component passing `do/end` blocks, you will have to explicitly handle the
-  assigns by giving it a clause:
+  component passing a `do/end` block, you will have to explicitly handle the
+  assigns by giving it a function clause:
```

Comment:
It is not really a function clause but I am not sure a better approach. Maybe we could say:

```suggestion
  assigns by giving it a clause by using `->`:
```

?

Reply:
Ah, right, it's a clause in the wider sense as seen in e.g. a case statement.

Maybe "by giving it a `->` clause" then?

Reply:
:+1:

Reply:
We should change this one too: https://github.com/phoenixframework/phoenix_live_view/blob/78938dcd42d245251ac7285b2d3c855795534b09/lib/phoenix_live_view/helpers.ex#L241